source/blog/2017-09-11-rspec-3-book-released.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Effective Testing With RSpec 3 has been released!
+author: Myron Marston
+---
+
+As we [announced back in February](/blog/2017/02/now-in-beta-effective-testing-with-rspec-3/),
+Ian Dees and I have written a new book for the Pragmatic Bookshelf
+about RSpec, entitled _Effective Testing with RSpec 3: Build Ruby Apps with Confidence_.
+We're happy to announce that the book is out of beta and is now shipping from
+[pragprog.com](https://pragprog.com/book/rspec3/effective-testing-with-rspec-3) and from
+[amazon.com](https://www.amazon.com/Effective-Testing-RSpec-Build-Confidence/dp/1680501984/).
+
+We put a ton of work over the last three years into this book, distilling our
+combined 20 years of testing experience into a book that explains how to do
```

Comment:
We use "book" twice here; any chance we could rephrase; maybe something like: 

> We put a ton of work over the last three years into this _project_, distilling our combined 20 years of testing experience into a book that explains how to do

Reply:
:+1: